sources/packages/backend/libs/services/src/queue/queue.service.ts

```diff
@@ -79,6 +80,16 @@ export class QueueService {
     return config;
   }
 
+  /**
+   * Get the advanced queue settings.
+   * @param queueName queue name.
+   * @returns queue advanced settings.
+   */
+  async getQueueSetting(queueName: QueueNames): Promise<AdvancedSettings> {
+    const queueConfig = await this.queueConfigurationDetails(queueName);
```

Comment:
`queueConfigurationDetails` is cached once requested for the first time, hence data is retrieved once.